tests/smoke/copy_test.go

```diff
@@ -21,6 +20,10 @@ import (
 // Stop the temporary registry
 // Copy the bundle to another location, this will fail unless we are properly using the relocation map
 func TestCopy(t *testing.T) {
+	test, err := tester.NewTest(t)
```

Comment:
This test was failing and preventing me from merging this PR (even though it's unrelated) because it's a flake #1928